packages/opentelemetry-sdk-trace-base/src/Tracer.ts

```diff
@@ -212,6 +214,11 @@ export class Tracer implements api.Tracer {
     return api.context.with(contextWithSpanSet, fn, undefined, span);
   }
 
+  /** Returns the active {@link GeneralLimits}. */
+  getGeneralLimits(): GeneralLimits {
```

Comment:
Why do we have this function? It is not part of the interface. If it is just for testing, you can access the private field by doing `tracer["_generalLimits"]` in the test.

Reply:
We can access the way you mentioned.

I just followed the pattern we already have like `getSpanLimits(): SpanLimits {` in line `223`